### DISCUSSION TITLE
practice-exam/practice-exam-2

### DISCUSSION THREAD

**Discussion by giscus** [kananinirav/AWS-Certified-Cloud-Practitioner-Notes#36] [Comments]
# practice-exam/practice-exam-2

If you are preparing for AWS Cloud Practitioner Exam, this guide will help you with quick revision before the exam. I share the notes I used to study and pass exam.

https://kananinirav.com/practice-exam/practice-exam-2.html

**Comment by ckearl** [comment]
The answer for Question 9 is outdated, as Amazon Connect replaced Infrastructure Event Management. The Enterprise plans offer Amazon Connect Premium.

**Reply by karry-lemon** [reply]
> The answer for Question 9 is outdated, as Amazon Connect replaced Infrastructure Event Management. The Enterprise plans offer Amazon Connect Premium.

I guess IEM was replaced by AWS Countdown

**Comment by reenuhaa** [comment]
Yes it is replaced by "AWS Countdown Premium"  service
Thank you for the info..

**Comment by Zehco** [comment]
29. Creating hypervisors? I can still create hypervisors myself on EC2. Isn't A, E?

**Reply by Zehco** [reply]
I can't believe 33 is BC. A-E all can do in transitional data center, as long as you have money.

**Reply by ajitsd** [reply]
"Traditional" assumes that there is a single data center. Agreed that it can still be achieved in traditional data centers with higher investment to achieve parity with AWS Cloud data centers.

**Reply by TommyPetersen23** [reply]
We had virtual distributed resources with fail-over, so not sure how #33 is even answered.

**Comment by ManuelRUhlig** [comment]
33. What are two advantages of using Cloud Computing over using traditional data centers? (Choose TWO)

    A. Reserved Compute capacity.
    B. Eliminating Single Points of Failure (SPOFs).
    C. Distributed infrastructure.
    D. Virtualized compute resources.
    E. Dedicated hosting.

shouldn't that be C+D instead of C+B?
Eliminating Single Points of Failure can be done in cloud but it depends on how you build it, while D is always true for cloud

**Reply by yesh773** [reply]
virtualization itself is not unique to cloud computing. Traditional on-premises data centers can also run virtual machines using technologies like VMware, Hyper-V, or KVM
